README.md

```diff
@@ -44,7 +44,271 @@ To build, ensure docker is installed, and run:
+Labels:        <none>
+Annotations:   pv.kubernetes.io/bind-completed: yes
+               pv.kubernetes.io/bound-by-controller: yes
+               storageClassName: glustervirtblock-csi
+               volume.beta.kubernetes.io/storage-provisioner: org.gluster.glustervirtblock
+Finalizers:    [kubernetes.io/pvc-protection]
+Capacity:      100Mi
+Access Modes:  RWO
+VolumeMode:    Filesystem
+Events:
+  Type       Reason                 Age    From                                                                                                        Message
+  ----       ------                 ----   ----                                                                                                        -------
+  Normal     ExternalProvisioning   2m34s  persistentvolume-controller                                                                                 waiting for a volume to be created, either by external provisioner "org.gluster.glustervirtblock" or manually created by system administrator
+  Normal     Provisioning           2m34s  org.gluster.glustervirtblock_csi-provisioner-glustervirtblockplugin-0_494f6242-1ee8-11e9-b6a0-0a580ae9416b  External provisioner is provisioning volume for claim "default/glusterblock-csi-pv"
+  Normal     ProvisioningSucceeded  2m34s  org.gluster.glustervirtblock_csi-provisioner-glustervirtblockplugin-0_494f6242-1ee8-11e9-b6a0-0a580ae9416b  Successfully provisioned volume pvc-1048edfb-1f06-11e9-8b7a-525400491c42
+Mounted By:  <none>
+```
+
+Verify gluster virtual block PV details:
+
+```
+[root@localhost]# kubectl describe pv
+Name:            pvc-1048edfb-1f06-11e9-8b7a-525400491c42
+Labels:          <none>
+Annotations:     pv.kubernetes.io/provisioned-by: org.gluster.glustervirtblock
+Finalizers:      [kubernetes.io/pv-protection]
+StorageClass:    glustervirtblock-csi
+Status:          Bound
+Claim:           default/glusterblock-csi-pv
+Reclaim Policy:  Delete
+Access Modes:    RWO
+VolumeMode:      Filesystem
+Capacity:        100Mi
+Node Affinity:   <none>
+Message:
+Source:
+    Type:              CSI (a Container Storage Interface (CSI) volume source)
+    Driver:            org.gluster.glustervirtblock
+    VolumeHandle:      pvc-1048edfb-1f06-11e9-8b7a-525400491c42
+    ReadOnly:          false
+    VolumeAttributes:      glusterbkpservers=gluster-kube1-0.glusterd2.gcs:gluster-kube2-0.glusterd2.gcs
+                           glusterserver=gluster-kube3-0.glusterd2.gcs
+                           glustervol=block_hosting_volume_ddd7ced7-7766-4797-9214-01fa9587472a
+                           storage.kubernetes.io/csiProvisionerIdentity=1548231861262-8081-org.gluster.glustervirtblock
+Events:                <none>
+```
+
+### Create a pod with RWO pvc claim
+
+```
+[root@localhost]# cat app.yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gluster-0
+  labels:
+    app: smallfile
+spec:
+  containers:
+  - name: smallfile
+    image: quay.io/ekuric/smallfile:latest
```

Comment:
what this container image does, cant we use existing redis image?

Reply:
Done